components/CommentModal/CommentModal.js

```diff
@@ -60,35 +66,77 @@ My thoughts:
       ? "oppose"
       : "have thoughts on"
 
+  const emailSuffix = `See more testimony on this bill at ${webSiteBillAddress}${bill.BillNumber}`
```

Comment:
consider moving the email text logic into separate functions -- this render function is getting quite large

Reply:
good idea.  I am looking into this.